src/main/resources/openapi-config.yaml

```diff
@@ -0,0 +1,14 @@
+resourceClasses:
+  - uk.gov.pay.webhooks.webhook.resource.WebhookResource
+  - uk.gov.pay.webhooks.healthcheck.HealthCheckResource
+readAllResources: true
+sortOutput: true
+prettyPrint: true
+
+openAPI:
+  info:
+    description: 'Webhooks API'
+    title: 'Webhooks API'
+    version: 'v1.0.0'
+  servers:
+    - url: 'http://webhooks_url'
```

Comment:
Is this field mandatory? If not could we remove it?

Reply:
Not mandatory. Removed